main.cpp

```diff
@@ -194,22 +241,25 @@ struct VaccumCleaner
     std::string informUserWhenFull();
 };
 
-/*
-Thing 3) Bicycle
-5 properties:
-    1) number of gears (int)
-    2) type (std::string)
-    3) speed (int)
-    4) cadence (std::string)
-    5) color (std::string)
-3 things it can do:
-    1) cycle at a certain speed
-    2) turn left or right
-    3) break
- */
+void VaccumCleaner::dustCollect(int dustQuantity, int time)
+{
```

Comment:
This function name does not conform with the following:
Name things what they are or what they do. 
the verb here is `collect`, but your function name doesn't put the words in the correct order. 
it should be `collectDust()`
- [ ] revise